_includes/templates/install/docker-install-note.md

```diff
@@ -0,0 +1,5 @@
+{% capture install_docker_user_to_group %}
+**NOTE: When you install docker, don't forget add your linux user to docker group**
```

Comment:
Don't forget to add your linux user to the docker group. See [Manage Docker as a non-root user](https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user).

Reply:
done